packages/server/src/externalTypes/requestPipeline.ts

```diff
@@ -121,3 +121,15 @@ export type GraphQLRequestContextDidEncounterSubsequentErrors<
 export type GraphQLRequestContextWillSendSubsequentPayload<
   TContext extends BaseContext,
 > = GraphQLRequestContextWillSendResponse<TContext>;
+
+export type GraphQLExecutionResult = {
```

Comment:
Do we need a new type here, or can we just use `ExecutionResult` from `graphql` (which is the type of `singleResult` where we assign it below)?

Reply:
Good call. Updated!